src/app/[locale]/incident/components/IncidentFormFooter.tsx

```diff
@@ -6,6 +6,8 @@ import { useTranslations } from 'next-intl'
 import { Button } from '@/components/ui/Button'
 import { useStepperStore } from '@/store/store'
 import { steps, usePathname as usePath, useRouter } from '@/routing/navigation'
+import { Paragraph } from '@utrecht/component-library-react/dist/css-module'
+import '@utrecht/design-tokens/dist/index.css'
```

Comment:
- De design tokens import is alleen nodig in `layout.tsx`, verder nergens
- `Paragraph` is ongebruikt. Later even kijken of er een linter is die dit kan herkennen.

Reply:
https://github.com/sweepline/eslint-plugin-unused-imports Een tijdje terug heb ik hier is naar gekeken. Doet volgens mij wel wat wij willen.